<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printskyDialogForm</class>
 <widget class="QWidget" name="printskyDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
     <property name="title" stdset="0">
      <string>PrintSky Plug-in Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Contents">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="Tabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="General">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="2" column="0" colspan="3">
              <widget class="QFrame" name="buttonsFrame">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="previewSkyPushButton">
                  <property name="text">
                   <string>Preview</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="printSkyPushButton">
                  <property name="text">
                   <string>Print</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QGroupBox" name="orientationGroupBox">
               <property name="title">
                <string>Orientation</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="orientationPortraitRadioButton">
                  <property name="text">
                   <string>Portrait</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="orientationLandscapeRadioButton">
                  <property name="text">
                   <string>Landscape</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QCheckBox" name="invertColorsCheckBox">
                 <property name="text">
                  <string>Invert colors</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="scaleToFitCheckBox">
                 <property name="text">
                  <string>Scale to fit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="printDataCheckBox">
                 <property name="text">
                  <string>Print scene information</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="selectedObjectCheckBox">
                 <property name="text">
                  <string>Print selected object information</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="printSSEphemeridesCheckBox">
                 <property name="text">
                  <string>Print Solar System ephemerides</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="magLimit_horizontalLayout">
                 <item>
                  <widget class="QCheckBox" name="limitMagnitudeCheckBox">
                   <property name="toolTip">
                    <string>Don't show data for objects dimmer than...</string>
                   </property>
                   <property name="text">
                    <string>Limit magnitude</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="limitMagnitudeDoubleSpinBox">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="minimum">
                    <double>-3.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>25.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="About">
         <attribute name="title">
          <string>About</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="labelAbout">
            <property name="text">
             <string>This plugin allows the user to preview and print the current view and some aditional data as the current location, the screen time adding the GMT difference, and also a radius/magnitude scale as a field reference.

In a optional second page you can get some Solar System ephemerides.</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Author: &lt;a href=&quot;mailto:maslarocaxica@gmail.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Pep Pujols&lt;/span&gt;&lt;/a&gt; (with fixes by Bogdan Marinov, and port to Qt5 by Georg Zotti)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Tabs</tabstop>
  <tabstop>invertColorsCheckBox</tabstop>
  <tabstop>scaleToFitCheckBox</tabstop>
  <tabstop>printDataCheckBox</tabstop>
  <tabstop>printSSEphemeridesCheckBox</tabstop>
  <tabstop>orientationPortraitRadioButton</tabstop>
  <tabstop>orientationLandscapeRadioButton</tabstop>
  <tabstop>previewSkyPushButton</tabstop>
  <tabstop>printSkyPushButton</tabstop>
  <tabstop>labelAbout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
